### PR TITLE
Add task to regenerate cassette

### PR DESCRIPTION
**Pull request by djberg96**
This is a handy rake task to regenerate the VCR cassette when needed. It effectively replaces the old `generate_data.rb` file.

### CODE REVIEW

**Comment by djberg96** [comment]
@durandom Ok, I moved it.

File: lib/tasks_private/scvmm.rake [hunk 1]
```diff
@@ -0,0 +1,54 @@
+require 'winrm'
+require 'json'
+require 'yaml'
+
+namespace :manageiq do
+  namespace :providers do
+    namespace :scvmm do
```
Comment (durandom) [inline lib/tasks_private/scvmm.rake]: I put a private task in the aws provider at top level namespace, see [here](https://github.com/ManageIQ/manageiq-providers-amazon/blob/087f2bb574304ce915945425f8e91d0252fc492a/lib/tasks_private/instance_types.rake)

The `manageiq:providers` namespace was useful when the tasks where included also in the core app, but thats no longer the case.

Up to you to leave it namespaced or not

File: lib/tasks_private/scvmm.rake [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      desc "Regenerate VCR cassette"
+      task :regenerate_cassette, [:username, :password, :host, :port] do |_t, args|
+        secrets = Rails.application.secrets.scvmm
```
Comment (durandom) [inline lib/tasks_private/scvmm.rake]: I see you have the same call in azure. Does this work? Where do  the secrets come from? `spec/mangeiq/config/secrets.yml` ? Or does `<gem_root>/config/secrets.yml` work too?

File: lib/tasks_private/scvmm.rake (hunk 1 above)
Comment (djberg96) [inline lib/tasks_private/scvmm.rake]: Ok, I've removed the outer namespace and just used `scvmm`.

File: lib/tasks_private/scvmm.rake (hunk 2 above)
Comment (djberg96) [inline lib/tasks_private/scvmm.rake]: They come from `spec/manageiq/config/secrets.yml`. The last time I tried it under `<gem_root>/config/secrets.yml` it didn't work.

Comment (durandom) [inline lib/tasks_private/scvmm.rake]: too bad.

**Pull request merged**